addons/mail/static/src/core/web/chatter.xml

```diff
@@ -72,7 +72,7 @@
                 <t t-if="state.composerType === 'message' and state.thread.followers.size > 0">
                     <div class="flex-shrink-0 px-3 pt-3 text-truncate small mb-2" style="margin-left:48px">
                         <span class="fw-bold">To:</span> <t t-out="toRecipientsText"/>
-                        <button class="o-mail-Chatter-recipientListButton btn btn-link badge rounded-pill border-0 p-1 ms-1" title="Show all recipients" t-on-click="onClickRecipientList">
+                        <button t-if="state.thread.recipients.size > 0" class="o-mail-Chatter-recipientListButton btn btn-link badge rounded-pill border-0 p-1 ms-1" title="Show all recipients" t-on-click="onClickRecipientList">
```

Comment:
Shouldn't this condition be replaced by `state.thread.followers.size` above?

Reply:
sur!